controls/srg_ctr/SRG-APP-000068-CTR-000120.yml

```diff
@@ -10,3 +10,116 @@ controls:
   - oauth_login_template_set
```

Comment:
Please remove the `classification_banner` and `oauth_login_template_set` rules from the control file. The reason is that `banner_or_login_template_set` is a meta-rule that just OR-s the two, so it's redundant to add them explicitly but also would make generating the spreadsheet from the controls file confusing.

Reply:
btw see `applications/openshift/general/banner_or_login_template_set/oval/shared.xml` for how the rule is implemented